spec/javascripts/helpers/markdown_editor_spec.js

```diff
@@ -0,0 +1,275 @@
+describe("Diaspora.MarkdownEditor", function() {
+  beforeEach(function() {
+    spec.content().html("<textarea id='fake-textarea'></textarea>");
+    this.$el = $("#fake-textarea");
+    Diaspora.I18n.language = "en";
+    var locale = {
+      // Localisation
+      /* eslint-disable camelcase */
+      markdown_editor: {
+        preview: "Preview",
+        write: "Write",
+        tooltips: {
+          bold: "Bold",
+          italic: "Italic",
+          heading: "Heading",
+          insert_link: "Insert link",
+          insert_image: "Insert image",
+          insert_ordered_list: "Insert ordered list",
+          insert_unordered_list: "Insert unordered list",
+          preview: "Preview message",
+          write: "Edit message",
+          cancel: "Cancel message",
+          quote: "Insert quotation",
+          code: "Insert code"
+        },
+        texts: {
+          strong: "strong text",
+          italic: "italic text",
+          heading: "heading text",
+          insert_link_description_text: "enter link description here",
+          insert_link_help_text: "Insert link here",
+          insert_image_description_text: "enter image description here",
+          insert_image_help_text: "Insert image link here",
+          insert_image_title: "enter image title here",
+          list: "list text here",
+          quote: "quotation text here",
+          code: "code here"
+        }
+      }
+      /* eslint-enable camelcase */
+    };
+    Diaspora.I18n.load(locale, "en", locale);
```

Comment:
Regenerate fixtures, then you shouldn't need to set the locales manually: https://github.com/diaspora/diaspora/blob/develop/spec/javascripts/jasmine_helpers/SpecHelper.js#L161